nml/actions/action5.py

```diff
@@ -64,7 +64,7 @@ class Action5BlockType:
     "TTDP_GUI_25": (0x07, 73, Action5BlockType.FIXED),
     "TTDP_GUI": (0x07, 93, Action5BlockType.FIXED),
     "CANALS": (0x08, 65, Action5BlockType.OFFSET),
-    "ONE_WAY_ROAD": (0x09, 6, Action5BlockType.OFFSET),
+    "ONE_WAY_ROAD": (0x09, 18, Action5BlockType.OFFSET),
```

Comment:
Specs say 12 sprites, what is correct?

Reply:
18 is correct.

the commit says "Action5 type 0x09 is extended to accept a set of 18 sprites."

the specs say "The 6 sprites originally supported from (0.6|2.5) are, in order: [...] From (13|x) an *additional* 12 sprites are supported"

(my emphasis, perhaps it can be clearer)

Reply:
I've edited https://newgrf-specs.tt-wiki.net/wiki/Action5#09_One-way_road_arrows to hopefully be clearer.

Reply:
Might I suggest adding the example spritesheet from this PR to that section to help make the order of sprites clearer? Also would like to politely remind that the NML docs need updating as well 🙂 Otherwise I've already used this PR in the last release of RattRoads and can confirm it works as intended.

Reply:
@Andrew350 It's a wiki :) You do not have to wait for some volunteer to expand the documentation, you can do it as well.